src/python/pants/core/goals/test.py

```diff
@@ -316,6 +324,17 @@ def register_options(cls, register) -> None:
                 "system supports this."
             ),
         )
+        register(
+            "--extra-env-vars",
+            type=list,
+            member_type=str,
+            default=[],
+            help="Specify pairs of ENV_NAME=value to be included in the environment of test processes.",
```

Comment:
You don't have to give the value, right? It can be just the `ENV_NAME`, which means it will be read from the parent Pants process?

Reply:
I'm assuming Eric is suggesting to update the help text to reflect this.